src/main/java/com/sap/oss/phosphor/fosstars/data/owasp/experimental/VulnerabilitiesFromOwaspDependencyCheck.java

```diff
@@ -0,0 +1,404 @@
+package com.sap.oss.phosphor.fosstars.data.owasp.experimental;
+
+import static com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures.VULNERABILITIES;
+import static com.sap.oss.phosphor.fosstars.model.other.Utils.setOf;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sap.oss.phosphor.fosstars.data.DataProvider;
+import com.sap.oss.phosphor.fosstars.data.NoValueCache;
+import com.sap.oss.phosphor.fosstars.data.UserCallback;
+import com.sap.oss.phosphor.fosstars.data.ValueCache;
+import com.sap.oss.phosphor.fosstars.model.Feature;
+import com.sap.oss.phosphor.fosstars.model.ValueSet;
+import com.sap.oss.phosphor.fosstars.model.owasp.experimental.Dependency;
+import com.sap.oss.phosphor.fosstars.model.owasp.experimental.OwaspDependencyCheckEntry;
```

Comment:
If I understand correctly, this class and other classes in that package were generated from JSON schema. Is that correct?

If so, please move them to `com.sap.oss.phosphor.fosstars.data.owasp.experimental.data` package. `com.sap.oss.phosphor.fosstars.model` is not a right place for this unless you use these classes in feature values.

Please use `@JsonIgnoreProperties` annotation to specify fields that you don't currently use. That would reduce amount of memory that is required for the provider. Example:

https://github.com/SAP/fosstars-rating-core/blob/master/src/main/java/com/sap/oss/phosphor/fosstars/nvd/data/CVE.java

Please try to reduce amount of code in this generated classes. For example, I am not sure that you really need generated `toString()` methods. Or, remove unused gettters/setters (for example, for fields that were added to `@JsonIgnoreProperties`).

It would be also good if you added a package info file that describes how the classes were generated.